### PR TITLE
Nonres hhgf

### PR DESCRIPTION
**Pull request by acarvalh**
Adding again MG cards
@OlivierBondu

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/HH/GF_HH_BM/GF_HH_1/GF_HH_1_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+ False        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.188  = scale            ! fixed ren scale
+ 91.188  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.188  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ -1 = dynamical_scale_choice ! Select one of the preselect dynamical choice
+ 1.0  = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Matching - Warning! ickkw > 1 is still beta
+#*********************************************************************
+ 1 = ickkw            ! 0 no matching, 1 MLM, 2 CKKW matching
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/HH/GF_HH_BM/GF_HH_1/GF_HH_1_run_card.dat]: from what I see you don't have matching (i.e. no additional jets) so maybe this needs to go to 0 ?

Comment (acarvalh) [reply bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/HH/GF_HH_BM/GF_HH_1/GF_HH_1_run_card.dat]: It is done, together with update PDF set

**Comment by perrozzi** [comment]
seems ok to me, let's wait for Olivier's comments

**Comment by OlivierBondu** [comment]
Looks good to me, thanks!
One small thing maybe: could you add a comment in the various `customizecards.dat` about which bsm parameter corresponds to which coupling ? This would probably help for bookkeeping in the future

**Comment by OlivierBondu** [comment]
Hi ! is there anything preventing to merge this one ? Let us know

**Pull request merged**